e2e-tests/micronaut-patio-api-version/src/test/java/org/evomaster/e2etests/micronaut/patio/TcpPortMicronautPatioEMTest.java

```diff
@@ -0,0 +1,67 @@
+package org.evomaster.e2etests.micronaut.patio;
+
+import com.foo.micronaut.patio.MicronautPatioTestController;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.evomaster.core.problem.rest.HttpVerb;
+import org.evomaster.core.problem.rest.RestIndividual;
+import org.evomaster.core.search.Solution;
+import org.evomaster.e2etests.utils.RestTestBase;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.core.Is.is;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@Disabled
+public class TcpPortMicronautPatioEMTest extends RestTestBase {
+
+    @BeforeAll
+    public static void initClass() throws Exception {
+        RestTestBase.initClass(new MicronautPatioTestController());
+    }
+
+    @Test
+    public void testRunEM() throws Throwable {
+
+        runTestHandlingFlaky("TcpPortMicronautPatioEMTest.java", "com.foo.TcpPortMicronautPatioEMTest.java", 100, false, (args) -> {
```

Comment:
the parameters eg, `com.foo.TcpPortMicronautPatioEMTest.java` might be incorrect, please check it a bit.